apps/web-giddh/src/app/all-modules/all-modules.component.html

```diff
@@ -0,0 +1,68 @@
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/']">Create Account</a></p>
+            </div>
+            <div class="col-md-3 col-sm-3 mb-5 col-xs-6">
+                <p>Invoice</p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/proforma-invoice/invoice/sales']">Sales Invoice</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/proforma-invoice/invoice/purchase']">Purchase Record</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/invoice/preview/credit']">Dr/Cr Note</a></p>
+                <p><a href="javascript:void(0)" class="coming-soon-report">Receipt Voucher <small class="italic">Coming soon</small></a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/invoice/preview/sales']">Invoice Management</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/purchase-management/purchase']">Purchase Management</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/expenses-manager']">Petty Cash Management</a></p>
+            </div>
+            <div class="col-md-3 col-sm-3 mb-5 col-xs-6">
+                <p>Inventory</p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/inventory']">Inventory</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/manufacturing/report']">Manufacturing</a></p>
+                <!-- <p><a href="javascript:void(0)" [routerLink]="['/pages/']">Job Work</a></p> -->
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/inventory/report']">Branch Transfer</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/settings/warehouse']">Warehouse</a></p>
+            </div>
+            <div class="col-md-3 col-sm-3 mb-5 col-xs-6">
+                <p>Security</p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/import/select-type']">Import Data</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/tallysync']">TALLY Import</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/company-import-export']">Company Import/Export</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/settings/permission']">Permission</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/settings/permission']">Integration</a></p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-3 col-sm-3 mb-5 col-xs-6">
+                <p>Analysis</p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/home']">Dashboard        </a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/reports/reports-dashboard']">Report</a></p>
+                <p><a href="javascript:void(0)" [routerLink]="['/pages/']">Sales Bifurcation</a></p>
```

Comment:
Please check the flow of this path what happens when User clicks on this link as it will not go anywhere

Reply:
URL added in Sales Bifurcation